BusBoard.ConsoleApp/Helpers.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using RestSharp;
+using RestSharp.Serialization.Json;
+
+namespace BusBoard.ConsoleApp
+{
+    class Helpers
+    {
+        public static List<BusEntry> GetBuses(string busStopCode, RestClient client)
+        {
+
+            var request = new RestRequest($"/StopPoint/{busStopCode}/Arrivals", DataFormat.Json);
+            var response = client.Execute<List<BusEntry>>(request);
+            var items = response.Data;
+            var sortedItems = items.OrderBy(o => o.expectedArrival).ToList();
```

Comment:
You can just return this. And you shouldn't need `toList` if you change the return type to `IEnumerable` 🙂 

Also, can you think of a more descriptive name than `o`? 🙂